.circleci/config.yml

```diff
@@ -23,12 +23,17 @@ jobs:
         type: string
     docker:
       - image: circleci/golang:<< parameters.go_version >>
+        user: root
     working_directory: /go/src/github.com/instana/go-sensor
     steps:
       - checkout
       - restore_cache:
           keys:
             - go-mod-v4-{{ checksum "go.sum" }}
+      - run:
+          command: apt-get --allow-releaseinfo-change-suite update -y || true
+      - run:
+          command: apt-get update && apt-get install ca-certificates libgnutls30 -y || true
```

Comment:
Can we combine these steps into one `run` command?

Reply:
yes, I will change it